ds_judgements_public_ui/templates/pages/home.html

```diff
@@ -5,6 +5,7 @@
         content="Judgments and decisions from 2003 onwards." />
 {% endblock meta_description %}
 {% block robots %}
+  <meta name="robots" content="nosnippet" />
```

Comment:
There might be value in allowing snippets of non-search results, but this is fine.

https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag#data-nosnippet-attr

Reply:
I considered this, but decided against it because only Google supports `data-nosnippet` on a page element level, but other engines like Bing support `nosnippet` at a page level.